go/ephemeral/user_ek.go

```diff
@@ -288,7 +298,39 @@ func getLatestUserEKMetadataMaybeStale(ctx context.Context, g *libkb.GlobalConte
 		return nil, err
 	}
 
-	return metadata, nil
+	return statement, nil
+}
+
+func getLatestUserEKMetadataMaybeStale(ctx context.Context, g *libkb.GlobalContext) (metadata *keybase1.UserEkMetadata, err error) {
+	defer g.CTrace(ctx, "getLatestUserEKMetadataMaybeStale", func() error { return err })()
+
+	statement, err := getLatestUserEKStatementMaybeStale(ctx, g)
+	if err != nil || statement == nil {
+		return nil, err
+	}
+
+	return &statement.CurrentUserEk, nil
+}
+
+func getAllActiveUserEKs(ctx context.Context, g *libkb.GlobalContext, now keybase1.Time) (usereks []keybase1.UserEkMetadata, err error) {
+	defer g.CTrace(ctx, "getActiveUserEKs", func() error { return err })()
+
+	statement, err := getLatestUserEKStatementMaybeStale(ctx, g)
+	if err != nil || statement == nil {
+		return nil, err
+	}
+
+	allExisting := append([]keybase1.UserEkMetadata{}, statement.ExistingUserEks...)
```

Comment:
this is why the statement should just be a list ;)

Reply:
I think it's a wash on convenience. Here we have to build the list, but in other places we'd have to say `statement.ExistingUserEKs[len(statement.ExistingUserEKs)-1]` instead of just `statement.CurrentUserEK`. I suppose in either case we could add a convenience function in extras. But probably this one line is the only place this concatenation will ever happen?

Reply:
mostly teasing for this specific instance